### PR TITLE
Update the wordpress example

### PR DESCRIPTION
**Pull request by yomovh**
* Add private network
* Update documentation to access to wordpress backoffice

### CODE REVIEW

File: use-cases/wordpress-k8s-mysql/README.md [hunk 1]
```diff
@@ -37,13 +37,14 @@ export OS_REGION_NAME="XX"
 
 ### Layer 1 : kube
 
-* Deployment of a managed Kubernetes cluster with its node-pool
+* Create a private network with a [gateway](https://www.ovhcloud.com/en-gb/public-cloud/gateway/)
+* Deployment of a managed Kubernetes cluster with its node-pool deployed on the private network
 
 ### Layer 2 : db & wordpress
```
Comment (scraly) [inline use-cases/wordpress-k8s-mysql/README.md:43]:
```suggestion
### Layer 2 : MySQL & Wordpress
```

File: use-cases/wordpress-k8s-mysql/README.md [hunk 2]
```diff
@@ -69,68 +70,79 @@ export OS_REGION_NAME="XX"
 
 ### Create the Terraform variables file (configuration)
 
-Create the Terraform variables configuration file and fill the needed informations.
+Create the 2 Terraform variables configuration files and fill the needed information.
```
Comment (scraly) [inline use-cases/wordpress-k8s-mysql/README.md:73]:
```suggestion
Two Terraform variables configuration files have been already created. Edit them to modify the needed information.
```

File: use-cases/wordpress-k8s-mysql/README.md [hunk 3]
```diff
@@ -69,68 +70,79 @@ export OS_REGION_NAME="XX"
...
 
 ```bash
-vi variables.tfvars
+vi variables_01.tfvars
+```
+Add the following for the kubernetes cluster & customize if needed
```
Comment (scraly) [inline use-cases/wordpress-k8s-mysql/README.md:78]:
```suggestion
The following variable allows to deploy the Kubernetes cluster in GRA11 region:
```

File: use-cases/wordpress-k8s-mysql/README.md [hunk 4]
```diff
@@ -69,68 +70,79 @@ export OS_REGION_NAME="XX"
...
+```
+kubernetes = {
+    region = "GRA11"
+}
+```
```
Comment (scraly) [inline use-cases/wordpress-k8s-mysql/README.md:84]:
```suggestion

If you want to deploy it in another region, edit the file with the wanted region.
```

File: use-cases/wordpress-k8s-mysql/README.md [hunk 5]
```diff
@@ -69,68 +70,79 @@ export OS_REGION_NAME="XX"
...
 
+Add the following & customize if needed
+```bash
+vim variables_02.tfvars
+```
+Add the following for the kubernetes cluster & customize if needed
```
Comment (scraly) [inline use-cases/wordpress-k8s-mysql/README.md:89]:
```suggestion
The following variable allows to deploy the MySQL DB v8 in GRA region with essential plan and db1-7 flavor:
```

File: use-cases/wordpress-k8s-mysql/README.md [hunk 6]
```diff
@@ -69,68 +70,79 @@ export OS_REGION_NAME="XX"
...
     plan         = "essential"
     flavor       = "db1-7"
     version      = "8"
-}
-
-kubernetes = {
-    region = "DE"
-}
+} 
 ```
```
Comment (scraly) [inline use-cases/wordpress-k8s-mysql/README.md:98]:
```suggestion

Customize the values if needed.

```

File: use-cases/wordpress-k8s-mysql/README.md [hunk 7]
```diff
@@ -69,68 +70,79 @@ export OS_REGION_NAME="XX"
...
-
-If you need to re-use the credentials in other scripts, you can export the user credentials and the URI
+### Login into Wordpress 
 
+Get the Load Balancer service IP
 ```bash
-export PASSWORD=$(terraform output -raw user_password)
-export USER=$(terraform output -raw user_name)
-export URI=$(terraform output -raw cluster_uri)
+kubectl --kubeconfig=./kubeconfig_file get svc
```
Comment (scraly) [inline use-cases/wordpress-k8s-mysql/README.md:131]:
```suggestion
kubectl --kubeconfig=./kubeconfig.yml get svc
```

File: .gitignore [hunk 8]
```diff
@@ -15,3 +15,5 @@ basics/utils/vars.csv
 
 containers-orchestration/managed-kubernetes/create-cluster-with-tf/my_kube_cluster.yml
 containers-orchestration/managed-private-registry/create-registry-with-pulumi/ovhcloud-registry-go/Pulumi.dev.yaml
+*.tfvars
+kubeconfig_file
```
Comment (scraly) [inline .gitignore:19]:
```suggestion
kubeconfig.yml
```

File: use-cases/wordpress-k8s-mysql/01-kube/main.tf [hunk 9]
```diff
@@ -1,33 +1,32 @@
...
+  min_nodes     = 3
 }
 
 # Create a local file for the kubeconfig file
 resource "local_file" "kubeconfig_file" {
-  depends_on = [
-    ovh_cloud_project_kube.wordpress_kube_cluster
-  ]
   content  = ovh_cloud_project_kube.wordpress_kube_cluster.kubeconfig
   filename = "kubeconfig_file"
```
Comment (scraly) [inline use-cases/wordpress-k8s-mysql/01-kube/main.tf:25]:
```suggestion
  filename = "kubeconfig.yml"
```

File: use-cases/wordpress-k8s-mysql/02-db-wordpress/provider.tf [hunk 10]
```diff
@@ -19,7 +19,7 @@ provider "ovh" {
 
 provider "helm" {
   kubernetes {
-    config_path = "kubeconfig_file"
+    config_path    = "kubeconfig_file"
```
Comment (scraly) [inline use-cases/wordpress-k8s-mysql/02-db-wordpress/provider.tf:22]:
```suggestion
    config_path    = "kubeconfig.yml"
```

File: use-cases/wordpress-k8s-mysql/README.md [hunk 11]
```diff
@@ -69,68 +70,79 @@ export OS_REGION_NAME="XX"
...
 ```
 
-With these exports you can go directly in any other example (e.g: go) to docker build and run it and see it working.
+The wordpress site is available at this IP. 
+
+The back office is available under `/wp-admin/`. The default user is `user`, the generated password can be retrieved using :
+```bash
+kubectl --kubeconfig=./kubeconfig_file get secret --namespace default wordpress -o jsonpath="{.data.wordpress-password}" | base64 -d
```
Comment (scraly) [inline use-cases/wordpress-k8s-mysql/README.md:138]:
```suggestion
kubectl --kubeconfig=./kubeconfig.yml get secret --namespace default wordpress -o jsonpath="{.data.wordpress-password}" | base64 -d
```

File: use-cases/wordpress-k8s-mysql/README.md [hunk 12]
```diff
@@ -69,68 +70,79 @@ export OS_REGION_NAME="XX"
...
+```
 
 ### Delete the DB and the cluster
```
Comment (scraly) [inline use-cases/wordpress-k8s-mysql/README.md:141]:
```suggestion
### Delete the DB and the Kubernetes cluster
```

**Review by scraly — CHANGES_REQUESTED** [review]

**Comment by yomovh** [comment]
I have made a change in the content by removing as suggested the dependency towards openstack provider. 

The requested changes have normally been adressed (since I updated the content, I may have missed some). 

Thanks for the review.

**Comment by philippart-s** [comment]
@yomovh can you clarify (by closing the request with a comment) when you don't want to apply @scraly's suggestion to have a more readable PR for people like me that comes at the end of the PR (this lets me know if the changes are completed or not).

The main point mentioned by @scraly was that the code wasn't executing, is that fixed with your updates?

**Comment by yomovh** [comment]
Hello,

@philippart-s 
It was already running fine on my machine ;-) though the pre requisites list was quite long. This is why I took the remark to remove the openstack provider dependency which should lead to an easier reproducibility.

As you suggested I also closed the comments that I took into account (see [my comment](https://github.com/ovh/public-cloud-examples/pull/44#issuecomment-2095616443))

**Comment by philippart-s** [comment]
thanks, it's easier for me to valid the PR 😉

**Review by philippart-s — APPROVED** [review]
🚀, @yomovh if it's ok for you I'll merge this PR

@scraly fell free to open issues if I missed some points 😉

**Pull request merged**